crates/bevy_ecs/src/system/commands.rs

```diff
@@ -417,9 +417,29 @@ impl<T: Component> Command for RemoveResource<T> {
 #[allow(clippy::float_cmp, clippy::approx_constant)]
 mod tests {
     use crate::{
+        component::{ComponentDescriptor, StorageType},
         system::{CommandQueue, Commands},
         world::World,
     };
+    use std::sync::{
+        atomic::{AtomicUsize, Ordering},
+        Arc,
+    };
+
+    #[derive(Clone, Debug)]
+    struct DropCk(Arc<AtomicUsize>);
+    impl DropCk {
+        fn new_pair() -> (Self, Arc<AtomicUsize>) {
+            let atomic = Arc::new(AtomicUsize::new(0));
+            (DropCk(atomic.clone()), atomic)
+        }
+    }
+
+    impl Drop for DropCk {
+        fn drop(&mut self) {
+            self.0.as_ref().fetch_add(1, Ordering::AcqRel);
```

Comment:
Since these are just tests and not performance critical, it's probably best to use `SeqCst` to avoid an possible weird atomic edge cases.

Reply:
I mean, it's an atomic counter, it could really be Relaxed here, as there are no dependencies across other loads and stores. AcqRel is plenty already. SeqCst is only ever useful if you need to guarantee sequential consistency between multiple atomic loads and stores that involve multiple memory addresses, which is not the case here.

Reply:
I mean, it's an atomic counter, it could really be Relaxed here, as there are no dependencies across other loads and stores. AcqRel is plenty already. SeqCst is only ever useful if you need to guarantee sequential consistency between multiple atomic loads and stores that involve multiple memory addresses, which is not the case here.

Reply:
Yea I was just confused as to why it was `AcqRel` since it doesn't really fit here. 
Generally I push for defaulting to `SeqCst` and then case by case choosing a differing ordering. So I would argue for chosing `Relaxed` since it makes the most sense here.

Reply:
Changed to `Relaxed`.